doc/classes/Input.xml

```diff
@@ -44,7 +45,7 @@
 			<return type="Vector3">
 			</return>
 			<description>
-				If the device has an accelerometer, this will return the movement.
+				If the device has an accelerometer, this will return the acceleration.
```

Comment:
Just gonna ask this for completions sake: What if it doesn't have an accelerometer, or doesn't have one that Godot can access? What will it return then?

Reply:
I think it returns `Vector3()`.